src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -132,7 +132,11 @@
   ([^Reader rdr]
    (read-record rdr nil))
   ([^Reader rdr add-fn]
-   (let [builder (-> (.readObject rdr) resolve deref)
+   (let [try-resolve #(if-let [clazz (resolve %)]
+                        clazz
+                        (throw (ex-info (str "Unable to resolve fact type symbol: '" % "'")
+                                        {:fact-type-class %})))
```

Comment:
I don't think this is specific to facts?  It is being used for all Clojure records from what I can see here (admittedly it has been a while since I read through this): https://github.com/cerner/clara-rules/blob/0.21.2/src/main/clojure/clara/rules/durability/fressian.clj#L386

Reply:
I think you are right. It really is a failure to resolve a record builder fn.

Reply:
corrected:
https://github.com/cerner/clara-rules/pull/476/commits/5252ca54994b2a521897bce2a6d8a2af5b223edf